src/cmdstan/print.cpp

```diff
@@ -105,8 +106,8 @@ int main(int argc, const char* argv[]) {
 
 
   // Prepare values
-  int n = 9;
-  
+  const int n = 9;
```

Comment:
[optional]
If you're setting up constant primitives, they can be declared as static.  It's not enough of a big deal in performance here to be a required change.

Reply:
Done.